### PR TITLE
Add Standard Ebooks feed to opdsbrowser.lua

### PR DESCRIPTION
**Pull request by QJKX**
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8606)

### CODE REVIEW

**Comment by QJKX** [comment]
Standard Ebooks is roughly a small subset of Project Gutenberg with better formatting.


https://standardebooks.org/about

>Standard Ebooks is a volunteer-driven effort to produce a collection of high quality, carefully formatted, accessible, open source, and free public domain ebooks that meet or exceed the quality of commercially produced ebooks. The text and cover art in our ebooks is already believed to be in the U.S. public domain, and Standard Ebooks dedicates its own work to the public domain, thus releasing the entirety of each ebook file into the public domain. All the ebooks we produce are distributed free of cost and free of U.S. copyright restrictions.

File: plugins/opds.koplugin/opdsbrowser.lua [hunk 1]
```diff
@@ -32,6 +32,10 @@ local CatalogCache = Cache:new{
 
 local OPDSBrowser = Menu:extend{
     opds_servers = G_reader_settings:readSetting("opds_servers", {
+        {
+            title = "Standard Ebooks",
```
Comment (Frenzie) [inline plugins/opds.koplugin/opdsbrowser.lua:36]: I think I'd prefer to keep Gutenberg at the top, with this one second. Didn't know they had OPDS, thanks! :-)

Comment (hius07) [reply plugins/opds.koplugin/opdsbrowser.lua:36]: You did actually, https://github.com/koreader/koreader/pull/8441.

Comment (Frenzie) [reply plugins/opds.koplugin/opdsbrowser.lua]: Heh, well, clearly I didn't try out.
 that specific feed. :-)

**Review by Frenzie — APPROVED** [review]

**Comment by NiLuJe** [comment]
Missing a onetimemigration block to update existing users ;).

**Comment by solarkite** [comment]
I’ve been curious about this feed. However, after checking in a few nightlys and now a stable, I still don’t see “standard ebooks”

**Comment by Frenzie** [comment]
Looks like a fix may be coming soon. :-)

**Comment by ryanwwest** [comment]
How exactly does this get past the username/password credential requirement for standard ebooks? KOReader obviously handles it no problem, but with other epub software or browsers the same url needs a username.

**Comment by Frenzie** [comment]
Are you thinking of a different website? https://standardebooks.org/ doesn't require any passwords.

**Comment by ryanwwest** [comment]
So when I visit this link: https://standardebooks.org/feeds/opds

I get prompted with a username and password box. Then if I ignore it, the page says the below, making me think OPDS isn't free, see last sentence especially (and it didn't work for me in Thorium reader OPDS registration) :

"Accessing the feeds
Our New Releases Atom/RSS ebook feeds are open to everyone.

You can get access to our other ebook feeds in one of the following ways:

Join the Patrons Circle by making a small donation in support of our mission. Patrons have full access to our ebook feeds for the duration of their gift.

... 

If you’re a Patrons Circle member, when prompted enter your email address and leave the password field blank to access an ebook feed. "

**Comment by Frenzie** [comment]
In that case maybe it's an issue on their end. We don't intend to bypass anything.

**Pull request merged**